build.rs

```diff
@@ -277,6 +279,7 @@ fn get_llvm_cflags() -> String {
         .join(" ")
 }
 
+#[cfg(not(feature = "no-llvm-linking"))]
```

Comment:
I think this would be better expressed as `if cfg!(feature = "no-llvm-linking") { return; }`, better capturing the intent of "do nothing if this feature is enabled."

Reply:
Thanks, that makes perfect sense! I'll update the commit.